src/Glpi/Asset/Capacity/IsInventoriableCapacity.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Asset\Capacity;
+
+use Glpi\Inventory\Inventory;
+use Session;
+
+class IsInventoriableCapacity extends AbstractCapacity
+{
+    public function getLabel(): string
+    {
+        return Inventory::getTypeName(Session::getPluralNumber());
+    }
+
+    public function getIcon(): string
+    {
+        return Inventory::getIcon();
+    }
+
+    public function getCapacityUsageDescription(string $classname): string
+    {
+        return sprintf(
+            __('Used by %1$s assets'),
+            $this->countAssets($classname)
```

Comment:
To be more precise, I think it should count the assets with the `is_dynamic=1` property. The `isUsed()` method should be overriden too to return false if no assets from the current definition are dynamic.

Reply:
Done. Was not entirely sure for the `isUsed()` part, seems there is no similar case existing.